cpp/tests/io/orc_test.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
Huh, we did not change the file in 2020 :P

Please add the negative test for write after close (unless I missed it).